airflow/models/taskinstance.py

```diff
@@ -790,11 +791,11 @@ def log_url(self):
         )
 
     @property
-    def mark_success_url(self):
+    def mark_success_url(self) -> str:
         """URL to mark TI success"""
         base_url = conf.get('webserver', 'BASE_URL')
-        return base_url + (
-            "/confirm"
+        return (
+            f"{base_url}/confirm"
```

Comment:
This avoids the "Optional" aspect of `conf.get`, plus I think it's easier to read.

Reply:
Should we use `get_mandatory_value` instead?

Reply:
We could, and it'll be functionally equivalent because we have a default. That said, I do still like the fstring approach better anyways.